src/main/scala/sigmastate/serialization/DataJsonEncoder.scala

```diff
@@ -0,0 +1,255 @@
+package sigmastate.serialization
+
+import java.math.BigInteger
+
+import io.circe._
+import io.circe.syntax._
+import org.ergoplatform.ErgoBox
+import org.ergoplatform.ErgoBox.NonMandatoryRegisterId
+import scalan.RType
+import scalan.RType.PairType
+import scorex.crypto.hash.Digest32
+import scorex.util._
+import sigmastate.Values.{Constant, EvaluatedValue}
+import sigmastate._
+import sigmastate.eval._
+import sigmastate.lang.SigmaParser
+import sigmastate.lang.exceptions.SerializerException
+import special.collection.{Coll, PairColl, collRType}
+import special.sigma._
+import spire.syntax.all.cfor
+
+import scala.collection.mutable
+
+object DataJsonEncoder {
+  def encode[T <: SType](v: T#WrappedType, tpe: T): Json = {
+    val encodedType = tpe.toTermString
+    val encodedData = encodeData(v, tpe)
+    Json.obj(
+      "type" -> Json.fromString(encodedType),
+      "value" -> encodedData,
+    )
+  }
+
+  private def encodeBytes: Encoder[Array[Byte]] = (bytes: Array[Byte]) => {
+    java.util.Base64.getEncoder.encodeToString(bytes).asJson
```

Comment:
@gostkin Why Base64 here? Ergo JSON API uses Base16. 
See - http://github.com/ScorexFoundation/sigmastate-interpreter/blob/cf570014ba16bad555a73d71d539568a4cf341a1/src/main/scala/org/ergoplatform/JsonCodecs.scala#L58-L58
Same for `decodeBytes`.